% min master timer Poulated

```diff
@@ -367,9 +369,11 @@ export function render(index) {
     //*************************************************
     // Color Fade Pulse Render, Slot 6 Mode 8
     //*************************************************
-    t1 = time(.01) // For hue movement
-    t2 = time(.02) // For pulse movement
-    t3 = time(.1)  // White / desaturation movement
+
+    // I commented this out because it's already beforeRender code in the render
+    // t1 = time(.01) // For hue movement
+    // t2 = time(.02) // For pulse movement
+    // t3 = time(.1)  // White / desaturation movement
```

Comment:
I commented out this section because it looks like some beforeRender code wound up in the render function